src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointTimelinePoint.js

```diff
@@ -50,32 +32,28 @@ function BreakpointTimelinePoint({
   seek,
 }) {
   const [leftPercentOffset, setLeftPercentOffset] = useState(0);
-  const classnameObj = getClassnameObject(point.point, executionPoint);
-
-  // Hide the point if there is no execution point, which happens in between pauses when using the
-  // command bar's resume button.
-  const opacity = executionPoint ? "1" : "0";
 
   useEffect(() => {
-    const offset = getLeftPercentOffset({
-      point,
-      timelineNode,
-      zoomRegion,
-      markerWidth: pointWidth,
-    });
-
-    setLeftPercentOffset(offset);
-  });
+    setLeftPercentOffset(
+      getLeftPercentOffset({
+        point,
+        timelineNode,
+        zoomRegion,
+        markerWidth: pointWidth,
+      })
+    );
+  }, [point, timelineNode, zoomRegion]);
 
   return (
     <div
-      className={classnames("breakpoint-navigation-timeline-point", classnameObj)}
+      className={classnames("breakpoint-navigation-timeline-point", {
+        past: BigInt(point.point) < BigInt(executionPoint),
+        future: BigInt(point.point) > BigInt(executionPoint),
+        pause: BigInt(point.point) == BigInt(executionPoint),
+      })}
       title={`${index + 1}/${analysisPoints.length}`}
       onClick={() => seek(point.point, point.time, true)}
-      style={{
-        left: `${leftPercentOffset}%`,
-        opacity: opacity,
```

Comment:
dropping opacity because I want to see the markers when the video is playing. I also think we should as a rule always show them...